theatre/studio/src/panels/SequenceEditorPanel/layout/tree.ts

```diff
@@ -22,6 +22,7 @@ export type SequenceEditorTree_Row<Type> = {
   depth: number
   top: number
   n: number
+  trackId?: string
```

Comment:
`trackId` would only be present when the row is a `SequenceEditorTree_PrimitiveProp` which itself does have a `trackId`, so this prop isn't necessary.